csv2dict/Csv2Dict.py

```diff
@@ -135,9 +136,10 @@ def get_existing_data(self, filepath, metadata_path):
         if self.blankout == True:
             return []
         else:
-            nx = utils.Nuxeo()
-            data = nx.get_metadata(path=filepath)
-            return data['properties']['ucldc_schema:{}'.format(metadata_path)]
+            if self.data == None:
+                nx = utils.Nuxeo()
+                self.data = nx.get_metadata(path=filepath)
+            return self.data['properties']['ucldc_schema:{}'.format(metadata_path)]
```

Comment:
So, adding a cache of the metadata. I take it nx.get_metadata is time consuming. Any danger of this causing memory use to build up?